scripts/init.sh

```diff
@@ -62,6 +62,7 @@ fi
 
 if [ ! -f "$STDOUT" ]; then
     mkdir -p $(dirname $STDOUT)
+    chown $USER:$GROUP $(dirname $STDOUT)
```

Comment:
Surely not. If `STDOUT` (and `STDERR`) are not changed from the defaults, this is changing the owner and group of `/dev`.